src/controller/response/balance-response.ts

```diff
@@ -62,6 +62,18 @@ export default interface BalanceResponse extends BaseUserResponse {
   lastTransferId: number;
 }
 
+/**
+ * @typedef TotalBalanceResponse
+ * @property {string} data.required - Date at which this total balance was calculated
+ * @property {number} totalPositive.required - The total amount of positive balance in SudoSOS
```

Comment:
why not a dinero object? seems more consistent with rest of monetairy responses.